CHANGELOG.md

```diff
@@ -49,18 +88,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (distribution-precompile) [#2055](https://github.com/evmos/evmos/pull/2055) Change the validator address in the events from string type to address type.
 - (recovery) [#2087](https://github.com/evmos/evmos/pull/2087) Remove `x/recovery` module.
 - (bank-precompile) [#2095](https://github.com/evmos/evmos/pull/2095) Add `bank` precompile.
-- (incentives) [#2070](https://github.com/evmos/evmos/pull/2070) Remove `x/incentives` module and burn incentives pool balance.
+- (incentives) [#2070](https://github.com/evmos/evmos/pull/2070) Remove `x/incentives` module and burn incentives pool balance (burning reverted in [#2131](https://github.com/evmos/evmos/pull/2131)).
```

Comment:
We actually kept the burning logic on the v16 logic. Should we keep the `(burning reverted in [#2131](https://github.com/evmos/evmos/pull/2131))`?

Reply:
sounds good, thanks for the suggestion

Reply:
```suggestion
- (incentives) [#2070](https://github.com/evmos/evmos/pull/2070) Remove `x/incentives` module and burn incentives pool balance.
```